libraries/AP_BattMonitor/AP_BattMonitor_INA2xx.cpp

```diff
@@ -42,8 +42,9 @@ extern const AP_HAL::HAL& hal;
 #define REG_238_MANUFACT_ID   0x3e
 #define REG_238_DEVICE_ID     0x3f
 
-// this should become a parameter in future
-#define MAX_AMPS 90.0
+#ifndef HAL_BATTMON_INA2XX_MAX_AMPS
```

Comment:
```suggestion
#ifndef DEFAULT_BATTMON_INA2XX_MAX_AMPS
```

would be more in-keeping with our new general schema for defines for parameter defaults.

But this isn't a blocker.

Reply:
changed